### DISCUSSION TITLE
adding ({datatables: true,}) to require('flowbite/plugin') is causing TypeError: require(...) is not a function

### DISCUSSION THREAD

**Discussion by anjanesh** [tailwindlabs/tailwindcss#14161] [Help]
I'm using tailwindcss-windows-x64.exe (v3.4.9 which I downloaded just now) - when I added `({datatables: true, })`
```
  plugins: [
      require('flowbite/plugin')({
      datatables: true,
    }),
    require('@tailwindcss/forms')
  ],
```

```
(env) PS D:\workspace\projectName\static> .\tailwindcss.exe -i ./src/input.css -o ./src/styles.css

Rebuilding...
TypeError: require(...) is not a function
    at D:\workspace\projectName\static\tailwind.config.js:60:33
    at evalModule (C:\snapshot\tailwindcss\node_modules\jiti\dist\jiti.js:1:256443)
    at jiti (C:\snapshot\tailwindcss\node_modules\jiti\dist\jiti.js:1:254371)
    at C:\snapshot\tailwindcss\lib\lib\load-config.js:56:30
    at loadConfig (C:\snapshot\tailwindcss\lib\lib\load-config.js:58:6)
    at Object.loadConfig (C:\snapshot\tailwindcss\lib\cli\build\plugin.js:135:49)
    at Object.getContext (C:\snapshot\tailwindcss\lib\cli\build\plugin.js:199:27)
    at C:\snapshot\tailwindcss\lib\cli\build\plugin.js:249:38
    at C:\snapshot\tailwindcss\lib\processTailwindFeatures.js:46:11
    at Object.Once (C:\snapshot\tailwindcss\lib\cli\build\plugin.js:257:19)
```

**Comment by florence0239** [comment]
Hello, @[myloweslife](https://www.my-loweslife.com)
It looks like you’re encountering an issue with the `require `function in your Tailwind CSS configuration. This error often occurs when the `require `function is not properly recognized or when the module being required is not correctly installed or referenced.

Here are a few steps you can try to resolve this issue:

Check the Installation: Ensure that `flowbite `and its dependencies are correctly installed. You can do this by running:
`npm install flowbite`
Update the Configuration: Make sure your `tailwind.config.js` file is correctly set up. It should look something like this:
`module.exports = {
  plugins: [
    require('flowbite/plugin')({
      datatables: true,
    }),
    require('@tailwindcss/forms'),
  ],
}`
Verify the Path: Ensure that the path to` flowbite/plugin` is correct and that the module is available in your `node_modules` directory.
Use Node.js: Since you’re using the standalone executable, consider switching to a Node.js environment to see if the issue persists. This can help isolate whether the problem is with the standalone executable or the configuration itself.
Check for Updates: Ensure you are using the latest version of Tailwind CSS and Flowbite. Sometimes, bugs are fixed in newer releases


Hope this will help you.
Best regards,
florence0239
